examples/wind/src/layers/particle-layer/particle-layer.js

```diff
@@ -167,7 +167,7 @@ export default class ParticleLayer extends Layer {
     });
 
     model.setAttributes({
-      posFrom: bufferTo
+      posFrom: bufferTo.setDataLayout({size: 4, instanced: 1})
```

Comment:
Q: Are we constantly resetting the layout to the same values? can this done during init?

Reply:
We are changing layout between TF and regular runs. In next luma PR I am actually adding a new method to update layout, which is lightweight than `setDataLayout`.